src/Modules/Station/Components/AddSong/PreviewVideo/PreviewVideo.tsx

```diff
@@ -39,7 +39,12 @@ export class PreviewVideo extends Component<any, any> {
   }
 
   public handleButtonAddSong() {
-    this.props.addSong(this.state.message);
+    const { message } = this.state;
+    this.props.addSong(message);
+
+    this.setState({
```

Comment:
To make it clear, you should write a function like resetMessage or clearMessage.
And it should be called after adding song successfully

Reply:
I edited it!